packages/api/src/utils/redis.ts

```diff
@@ -13,3 +13,5 @@ client.on('error', (err: any) => {
 });
 
 export default client;
+
+export { client as redisClient };
```

Comment:
I think this export is redundant. can import like
```
import redisClient from './utils/redis';
```

Reply:
Yeah I prefer the `import { redisClient }` version because it doesn't accidentally get renamed:

`import clickhouseClient from './utils/redis'; // no errors here` 

I've been bitten by that before :)

Reply:
Sounds good